File: app/src/main/java/com/my/televip/virtuals/androidx/LongSparseArray.java
package com.my.televip.virtuals.androidx;

import com.my.televip.obfuscate.ArgsResolver;
import com.my.televip.obfuscate.Obfuscate;

import java.util.ArrayList;

import de.robv.android.xposed.XposedHelpers;

public class LongSparseArray {

    Object longSparseArray;

    public LongSparseArray(Object longSparseArray) {
        this.longSparseArray = longSparseArray;
    }

    public ArrayList<Object> get(long id){
        return (ArrayList<Object>) XposedHelpers.callMethod(longSparseArray, Obfuscate.getMethodName("LongSparseArray", "get"), id);
    }
}

File: app/src/main/java/com/my/televip/virtuals/androidx/LongSparseArray.kt
package com.my.televip.virtuals.androidx

import com.my.televip.obfuscate.Obfuscate
import de.robv.android.xposed.XposedHelpers
import java.util.ArrayList

class LongSparseArray(@JvmField val longSparseArray: Any?) {

    @Suppress("UNCHECKED_CAST")
    fun get(key: Long): ArrayList<Any>? {
        val target = longSparseArray ?: return null
        return XposedHelpers.callMethod(target, Obfuscate.getMethodName("LongSparseArray", "get"), key) as? ArrayList<Any>
    }
}
